app/Phragile/MergedIntoTransaction.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Phragile;
+
+class MergedIntoTransaction extends Transaction {
+
+	const TYPE = 'mergedInto';
+
+	public function __construct($timestamp)
+	{
+		$this->timestamp = $timestamp;
+	}
+
+	public function getData()
```

Comment:
So I just stared at this class for 5 minutes (exaggerated) and had no idea what it's for and why we'd want to store MergedIntoTransactions without storing what got merged into what. I then dug into the code a bit deeper and remembered that merging a task is basically the same for Phragile as closing it.
Maybe we can add a little explanation either above the method or above the class to avoid future confusion . What do you think?